<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chitchat</class>
 <widget class="QMainWindow" name="Chitchat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chitchat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>1</y>
      <width>561</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>561</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sentBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/send.png</normaloff>:/images/send.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/exitLogin.png</normaloff>:/images/exitLogin.png</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFontComboBox" name="fontComboBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="fontSizeComboBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="boldBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ItalicBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/textitalic.png</normaloff>:/images/textitalic.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="underLineBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/textunder.png</normaloff>:/images/textunder.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="colorBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/textcolor.png</normaloff>:/images/textcolor.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showPanelBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/painting.png</normaloff>:/images/painting.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="expression">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/emoticon_smile.png</normaloff>:/images/emoticon_smile.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="picBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/insert-image.png</normaloff>:/images/insert-image.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="onLineUsersNum">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>500</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>在线人数:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>352</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/images/pic2.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <column>
        <property name="text">
         <string>图片</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>登录名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>主机名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ip地址</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
